src/plugins/intel_gpu/src/graph/include/primitive_inst.h

```diff
@@ -515,11 +521,11 @@ struct typed_primitive_impl : public primitive_impl {
         return execute_impl(event, reinterpret_cast<typed_primitive_inst<PType>&>(instance));
     }
 
-    std::vector<layout> get_internal_buffer_layouts() const override {
-        return get_internal_buffer_layouts_impl();
+    std::vector<BufferDescriptor> get_internal_buffer_descs(const kernel_impl_params& impl_params) const override {
+        return get_internal_buffer_descs_impl(impl_params);
     }
 
-    virtual std::vector<layout> get_internal_buffer_layouts_impl() const {
+    virtual std::vector<BufferDescriptor> get_internal_buffer_descs_impl(const kernel_impl_params&) const {
```

Comment:
Seems like there's no big value in this _impl method. Deriver classes may just override the original one.